app/components/forms/events/view/videoroom-form.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="ui grid">
     <div class="ui left aligned sixteen wide mobile eight wide computer column">
       <h3 class="ui header">
-        {{t 'Video Channel'}}
```

Comment:
The issue is about the admin section

Reply:
ok working on it

Reply:
@MeghalBisht

Reply:
No, this is not the file to be changed. The issue talks about the admin section.

Reply:
@MeghalBisht now see it

Reply:
@MeghalBisht if there is anything wrong in my approach then pls ma'am guide me.

Reply:
I dont think you are testing the changes. Still wrong. Run the project and use ember inspector to see the file to be changed